.github/e2e-tests.yml

```diff
@@ -991,6 +991,104 @@ runner-test-matrix:
       E2E_TEST_SELECTED_NETWORK: SIMULATED_1,SIMULATED_2
       E2E_JD_VERSION: 0.4.0
 
+  - id: smoke/ccip_rmn_test.go:^TestRMN_TwoMessagesOnTwoLanes$
+    path: integration-tests/smoke/ccip_rmn_test.go
+    test_env_type: docker
+    runs_on: ubuntu-latest
+    triggers:
+      - PR E2E Core Tests
+      - Merge Queue E2E Core Tests
+      - Nightly E2E Tests
+    test_cmd: cd integration-tests/smoke && go test -test.run ^TestRMN_TwoMessagesOnTwoLanes$ -timeout 12m -test.parallel=1 -count=1 -json
+    pyroscope_env: ci-smoke-ccipv1_6-evm-simulated
+    test_env_vars:
+      E2E_TEST_SELECTED_NETWORK: SIMULATED_1,SIMULATED_2
+      E2E_JD_VERSION: 0.4.0
+      E2E_RMN_RAGEPROXY_VERSION: master-5208d09
+      E2E_RMN_AFN2PROXY_VERSION: master-5208d09
+
+  - id: smoke/ccip_rmn_test.go:^TestRMN_MultipleMessagesOnOneLaneNoWaitForExec$
+    path: integration-tests/smoke/ccip_rmn_test.go
+    test_env_type: docker
+    runs_on: ubuntu-latest
+    triggers:
+      - PR E2E Core Tests
+      - Merge Queue E2E Core Tests
+      - Nightly E2E Tests
+    test_cmd: cd integration-tests/smoke && go test -test.run ^TestRMN_MultipleMessagesOnOneLaneNoWaitForExec$ -timeout 12m -test.parallel=1 -count=1 -json
+    pyroscope_env: ci-smoke-ccipv1_6-evm-simulated
+    test_env_vars:
+      E2E_TEST_SELECTED_NETWORK: SIMULATED_1,SIMULATED_2
+      E2E_JD_VERSION: 0.4.0
+      E2E_RMN_RAGEPROXY_VERSION: master-5208d09
+      E2E_RMN_AFN2PROXY_VERSION: master-5208d09
+
+  - id: smoke/ccip_rmn_test.go:^TestRMN_NotEnoughObservers$
+    path: integration-tests/smoke/ccip_rmn_test.go
+    test_env_type: docker
+    runs_on: ubuntu-latest
+    triggers:
+      - PR E2E Core Tests
+      - Merge Queue E2E Core Tests
+      - Nightly E2E Tests
+    test_cmd: cd integration-tests/smoke && go test -test.run ^TestRMN_NotEnoughObservers$ -timeout 12m -test.parallel=1 -count=1 -json
+    pyroscope_env: ci-smoke-ccipv1_6-evm-simulated
+    test_env_vars:
+      E2E_TEST_SELECTED_NETWORK: SIMULATED_1,SIMULATED_2
+      E2E_JD_VERSION: 0.4.0
+      E2E_RMN_RAGEPROXY_VERSION: master-5208d09
+      E2E_RMN_AFN2PROXY_VERSION: master-5208d09
+
+  - id: smoke/ccip_rmn_test.go:^TestRMN_DifferentSigners$
+    path: integration-tests/smoke/ccip_rmn_test.go
+    test_env_type: docker
+    runs_on: ubuntu-latest
+    triggers:
+      - PR E2E Core Tests
+      - Merge Queue E2E Core Tests
+      - Nightly E2E Tests
+    test_cmd: cd integration-tests/smoke && go test -test.run ^TestRMN_DifferentSigners$ -timeout 12m -test.parallel=1 -count=1 -json
+    pyroscope_env: ci-smoke-ccipv1_6-evm-simulated
+    test_env_vars:
+      E2E_TEST_SELECTED_NETWORK: SIMULATED_1,SIMULATED_2
+      E2E_JD_VERSION: 0.4.0
+      E2E_RMN_RAGEPROXY_VERSION: master-5208d09
+      E2E_RMN_AFN2PROXY_VERSION: master-5208d09
+
+#  - id: smoke/ccip_rmn_test.go:^TestRMN_NotEnoughSigners$
```

Comment:
I see a test like this in the repo, why do we comment it out?

Reply:
This is currently failing @dimkouv is working on a PR to fix this. After that we will uncomment this